webapp/channels/src/components/channel_header/channel_header.tsx

```diff
@@ -224,14 +226,17 @@ class ChannelHeader extends React.PureComponent<Props, State> {
 
         if (headerPopoverTextMeasurerRect && headerDescriptionRect) {
             if (headerPopoverTextMeasurerRect.width > headerDescriptionRect.width || headerText.match(/\n{2,}/g)) {
-                this.setState({showChannelHeaderPopover: true, leftOffset: this.headerDescriptionRef.current?.offsetLeft || 0});
+                const leftOffset = headerDescriptionRect.left - (this.props.hasMoreThanOneTeam ? 314 : 250);
```

Comment:
Did you get these numbers from somewhere in the code, or did you work them out yourself?

Reply:
Worked them out myself

Reply:
Okay, that's probably fine here. We generally try to avoid that sort of magic number, but there's enough things about this component (such as it using our ancient fork of React Bootstrap) that make it hard to get it to work without those.

Thanks for finding the right values to make it work!